packages/app/src/server/service/g2g-transfer.ts

```diff
@@ -18,9 +21,15 @@ import { G2GTransferError, G2GTransferErrorCode } from '../models/vo/g2g-transfe
 
 const logger = loggerFactory('growi:service:g2g-transfer');
 
+/**
+ * Header name for transfer key
+ */
 export const X_GROWI_TRANSFER_KEY_HEADER_NAME = 'x-growi-transfer-key';
 
-export const uploadConfigKeys = [
+/**
+ * Keys for file upload related config
+ */
+const uploadConfigKeys = [
```

Comment:
```suggestion
const UPLOAD_CONFIG_KEYS = [
```
file 内の global な const 定義の変数名なら大文字/snake_case の方がよいと思う。